__tests__/api.test.js

```diff
@@ -201,3 +201,45 @@ describe("POST /api/articles/:article_id/comments", () => {
       });
   });
 });
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("200: updates the vote count of an existing article with the given value", () => {
+    const body = {
+      inc_votes: 10,
+    };
+
+    return request(app)
+      .patch("/api/articles/1")
+      .send(body)
+      .expect(200)
+      .then((response) => {
+        expect(response.body).toHaveProperty("votes");
```

Comment:
We are testing a specific article, so we can be precise with the output we expect (to be an exact value). This also helps us ensure that the votes are actually being incremented